app/templates/teachers/resource-hub-view.jade

```diff
@@ -7,77 +7,124 @@ block content
   .container
 
     .content
-        h1(data-i18n="teacher.resource_hub")
+      if me.attributes.hourOfCode && me.isAnonymous()
+        #hour-of-code
+          h3
+            span(data-i18n="teacher.hoc_welcome")
+            | !
+          p
+            span(data-i18n="teacher.hoc_intro")
+            | :
+          ol.small
+            li
+              strong.spr(data-i18n="teacher.hoc_self_led") 
+              | - 
+              span.spl.spr(data-i18n="teacher.hoc_self_led_desc") 
+              | - 
+              a(href="/play/game-dev-hoc?hour_of_code=true")
+                span(data-i18n="teacher.hoc_game_dev")
+              span.spl.spr(data-i18n="teacher.hoc_and")
+              a(href="/play?hour_of_code=true") 
+                span(data-i18n="teacher.hoc_programming")
+              | .
+            li
+              strong.spr(data-i18n="teacher.hoc_teacher_led")
+              | - 
+              span.spr(data-i18n="teacher.hoc_teacher_led_desc1")
```

Comment:
This should have a .spl since the trailing whitespace on line 32 is likely to get trimmed at some point. Same with other similar places later on.